### PR TITLE
docs: updated all C components doc

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Documentation

#### What is expected?

To have better documentation for all starting C letter components.

⚠️ CreationProgress not include as full refacto is needed ⚠️ 

#### The following changes where made:

1. Updated stories

2. Fixed checkbox when progress is true not to be clickable

### CODE REVIEW

File: src/components/Checkbox/index.js [hunk 1]
```diff
@@ -93,7 +93,9 @@ const Checkbox = ({
           {...checkbox}
           hasChildren={hasChildren}
           size={size}
-          onChange={onChange}
+          onChange={e => {
```
Comment (chambo-e) [inline src/components/Checkbox/index.js:96]: Maybe add a test for this use case ?

**Review by DorianMaliszewski — CHANGES_REQUESTED** [review]
Differences between sidebar and story order/name

**Comment by DorianMaliszewski** [comment]
Capture d’écran 2021-06-01 à 11 54 53
Still there

**Review by DorianMaliszewski — CHANGES_REQUESTED** [review]
See comments

**Comment by matthprost** [comment]
> Capture d’écran 2021-06-01 à 11 54 53
> 
> Still there

Screenshot 2021-06-01 at 12 24 01

I think you may have cache

**Comment by DorianMaliszewski** [comment]
When clearing the cache and go to the chart it's still there. What did you have @chambo-e ?

**Review by chambo-e — APPROVED** [review]

**Review by DorianMaliszewski — APPROVED** [review]

**Pull request merged**